openreview/conference/builder.py

```diff
@@ -1533,6 +1534,28 @@ def is_release_authors(is_note_accepted):
             self.set_homepage_decisions(decision_heading_map=decision_heading_map)
         self.client.remove_members_from_group('active_venues', self.id)
 
+    def post_decisions(self, decisions_file):
+        with open(decisions_file) as file_handle:
+            decisions = [row for row in csv.reader(file_handle)]
+        for paper_id, decision in decisions:
+            paper_note = self.client.get_notes(id=paper_id)[0]
+            self.client.post_note(openreview.Note(
+                invitation='{}/Paper{}/-/Decision'.format(self.id, paper_note.number),
+                writers=[self.get_program_chairs_id()],
+                readers=[self.get_program_chairs_id(), self.get_senior_area_chairs_id(), self.get_area_chairs_id()],
```

Comment:
usually decision reader are the PCs and authors. You need to set the readers based on what the decision invitation says and the decision invitation is created based on the settings `released_to_authors`, etc.

I would move this code to the DecisionStage class if possible.